### PR TITLE
16.2.14 Release notes

### PR DESCRIPTION
**Pull request by yuriw**

### CODE REVIEW

**Comment by yuriw** [comment]
https://wip-yuriw-release-16.2.14-post-main.ceph.io/en/news/blog/2023/v16-2-14-pacific-released/

**Comment by ljflores** [comment]
Rendered here: https://wip-yuriw-release-16.2.14-post-main.ceph.io/en/news/blog/2023/v16-2-14-pacific-released/

File: src/en/news/blog/2023/v16-2-14-pacific-released/index.md [hunk 1]
```diff
@@ -0,0 +1,388 @@
...
+
+- tests: remove pubsub tests from multisite ([pr#48928](https://github.com/ceph/ceph/pull/48928), Yuval Lifshitz)
+
+- tools/ceph-dencoder: Fix incorrect type define for trash\_watcher ([pr#51778](https://github.com/ceph/ceph/pull/51778), Chen Yuanrun)
+
+- tools/ceph-kvstore-tool: fix segfaults when repair the rocksdb ([pr#51254](https://github.com/ceph/ceph/pull/51254), huangjun)
+
+- tools/cephfs-data-scan: support for multi-datapool ([pr#50523](https://github.com/ceph/ceph/pull/50523), Mykola Golub)
+
+- v16<span></span>.2<span></span>.13 ([pr#51404](https://github.com/ceph/ceph/pull/51404), Ceph Release Team, Patrick Donnelly, Rishabh Dave, Casey Bodley, Radoslaw Zarzynski, Mark Nelson)
```
Comment (ljflores) [inline src/en/news/blog/2023/v16-2-14-pacific-released/index.md:383]: I don't think we want this, right @neha-ojha?

Comment (ljflores) [reply src/en/news/blog/2023/v16-2-14-pacific-released/index.md:383]: Meaning line 383

Comment (yuriw) [reply src/en/news/blog/2023/v16-2-14-pacific-released/index.md]: @ljflores I removed it

**Comment by ljflores** [comment]
Added https://github.com/ceph/ceph/pull/53157 to the list of changes

**Review by ljflores — APPROVED** [review]

**Pull request merged**